app/assets/stylesheets/pages/login.scss

```diff
@@ -1,20 +1,23 @@
 ///login page styleing
-/// TODO: give card in desktop view consistent spacing
 .login {
   background: url("BackgroundImage.jpeg") no-repeat center center fixed;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
+
+    .mt-lg-5 {
+      margin-top: ($spacer * 8) !important;
```

Comment:
Ok i missed how this is utility is for responsive padding. However, `mt-lg-5` is already a thing, if I read that in the html code I would expect it to be spacer*3 because that's what it is in bootstrap. Its sort of like seeing 2+2=0 and then digging in and finding you overrode the + to mean - . You can just create your own class to accomplish the same thing.